docs/react-testing-library/api.md

```diff
@@ -254,3 +254,69 @@ This is a light wrapper around the
 [`react-dom/test-utils` `act` function](https://reactjs.org/docs/test-utils.html#act).
 All it does is forward all arguments to the act function if your version of
 react supports `act`.
+
+## `testHook`
+
+`testHook` is a utility to test custom hooks. It is designed to help test
+reusable hooks in isolation.
+
+You should also write integration tests for components using custom hooks, and
+one-off hooks should be tested as part of the component instead.
+
+**Usage**
+
+```jsx
+import { testHook } from 'react-testing-libary'
+
+testHook(hook[, renderOptions])
+```
+
+**Arguments**
+
+- `hook` customHook to test
+- `renderOptions` options object to pass to the underlying `render`. See
+  [render options](#render-options). This is mostly useful for wrapping the hook
```

Comment:
I guess you forgot to add this part where the link is leading? :)

Reply:
Need to implement the wrapper option for render

Reply:
Fixed

Reply:
I still don't see that link in this PR?

Reply:
Branch was out of date: https://deploy-preview-32--testing-library.netlify.com/docs/react-testing-library/api#render-options

Reply:
Well, that's rather misleading and people might think that `testHook` has same options as `render` which is not the case, right?

Reply:
It does now - just passes through

Reply:
Ah, I've missed that. Cool, that makes sense :)